pymatgen/cli/pmg_config.py

```diff
@@ -143,17 +143,17 @@ def setup_potcars(potcar_dirs: list[str]):
                     dest = os.path.join(base_dir, os.path.basename(fname))
                     shutil.copy(fname, dest)
                     ext = fname.split(".")[-1]
-                    if ext.upper() in ["Z", "GZ"]:
-                        with subprocess.Popen(["gunzip", dest]) as process:
+                    if ext.upper() in {"Z", "GZ"}:
+                        with subprocess.Popen(["/usr/bin/gunzip", dest]) as process:
```

Comment:
Commit 26ccb8591b3363783b15f224c6da09311f81d5cc need careful confirmation. 

[Ruff rule S607](https://docs.astral.sh/ruff/rules/start-process-with-partial-path/) prevents "partial executable path (relative path)" which might be hijacked by modifying the `PATH`.

However I'm concerned about the consistency of absolute paths for these binaries, as I'm not 100% sure they would reside in the same position across different distributions (not to mention different OS, like Ubuntu vs MacOS).